### PR TITLE
Update to mypy 0.770

### PR DESCRIPTION
**Pull request by intgr**
Mypy was updated to 0.770, see release notes: https://mypy-lang.blogspot.com/2020/03/mypy-0770-released.html

### CODE REVIEW

**Review by sobolevn — APPROVED** [review]
Thanks!

**Comment by intgr** [comment]
Beware, I haven't tested the mypy plugin yet, didn't expect it to be merged so fast :D

**Comment by sobolevn** [comment]
The CI passed, so I am confident that it works! 

Is there something I should reconsider? 🙂

**Comment by intgr** [comment]
Cool! I don't have any reason to expect breakage. I will test it with my project soon and let you know if there are any surprises.

File: scripts/enabled_test_modules.py [hunk 1]
```diff
@@ -198,7 +198,7 @@
     ],
     'from_db_value': [
         '"Cash" has no attribute',
-        'Argument 1 to "__str__" of "Decimal"',
+        '"__str__" of "Decimal"',
```
Comment (intgr) [inline scripts/enabled_test_modules.py:201]: Submitted PR to Django to fix their broken test code: https://github.com/django/django/pull/12726

**Pull request merged**